roles/awsenv/tasks/iam.yml

```diff
@@ -22,10 +22,10 @@
     policy_json: " {{ lookup('file', 'common.iam.policy') }} "
   with_items: "{{ vpc.iam_roles }}"
 
-- name: Create monitoring iam policy
-  iam_policy:
-    state: present
-    policy_name: 'monitoring-policy-{{ opg_data.stack }}'
-    iam_type: role
-    iam_name: 'monitoring-role-{{ opg_data.stack }}'
-    policy_json: " {{ lookup('file', 'monitoring.iam.policy') }} "
+#- name: Create monitoring iam policy
```

Comment:
If we're moving monitoring into it's own role, should we not delete this rather than comment it out